types/node_definition.go

```diff
@@ -289,6 +291,13 @@ func (n *NodeDefinition) GetSANs() []string {
 	return n.SANs
 }
 
+func (n *NodeDefinition) GetWaitFor() []string {
```

Comment:
add a comment?

Reply:
None of the Getters in that file has any comment. And it is basically just a getter...
https://github.com/srl-labs/containerlab/blob/1eb93a1382932905272e3a237120cf8d35dd3b91/types/node_definition.go#L294
If you want a comment, we should add that to all of the getters there